test/integration/dialects/sqlite/dao-factory.test.js

```diff
@@ -108,7 +108,7 @@ if (dialect === 'sqlite') {
             expect(user.name).to.equal('user');
           });
 
-          it.skip('should make aliased attributes available', async function () { // eslint-disable-line mocha/no-skipped-tests
+          it('should make aliased attributes available', async function () {
```

Comment:
Any idea why we skipped this test before?

Reply:
Not sure, I re-enabled it to see which dialects (if any) fails. Currently they all fail, I'll look into it

Reply:
Yeah but it doesn't fail because of this so think its good that you re-enabled it then

Reply:
Nevermind, it failed this time

Reply:
Apparently the test was disabled in https://github.com/sequelize/sequelize/commit/e10114fc58f4faaebc712d4711ecb8abf887a0b2 back in January 2014. Not sure why, but I think we can remove this test if we've been skipping this for this long

Reply:
Confirmed that it still fails

If you're still on a computer, would you mind doing it?

Reply:
Done

Reply:
Thanks!